setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = shieldapi
-version = v1.2.0
-author = sintef, Fraunhofer IWM
+version = v2.0.0
```

Comment:
This should not be done manually but with the bumpver cli. Since this makes the way of publishing safer and more automated.

Reply:
I see. I can revert it. What about the author? Should I change it manually?

Reply:
Yes, the author is not affected by bumpver.